frontend/src/core/config/config.ts

```diff
@@ -82,6 +82,10 @@ export const editorFontSizeAtom = atom<number>((get) => {
   return get(resolvedMarimoConfigAtom).display.code_editor_font_size;
 });
 
+export const localeAtom = atom<string | null | undefined>((get) => {
+  return get(resolvedMarimoConfigAtom).display.locale;
```

Comment:
Could be nice to normalize to `null` or `undefined` here for the none case?

```suggestion
  return get(resolvedMarimoConfigAtom).display.locale ?? undefined;
```